web/src/pages/Automations/Automations.css

```
@@ -1,8 +1,10 @@
 .automations-page {
+  /* Also read by the desktop shell's drag strip (App.css). */
+  --page-inset: clamp(16px, 3vw, 32px);
   display: flex;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)